Makefile

```
@@ -23,3 +23,26 @@ docker-build:
 .PHONY: offsets
 offsets:
 	cd offsets-tracker; OFFSETS_OUTPUT_FILE="../pkg/inject/offset_results.json" go run main.go
+
+.PHONY: install-tools
+install-tools:
+	go install github.com/google/go-licenses@latest
+
+.PHONY: update-licenses
+update-licenses: install-tools
+	rm -rf LICENSES; \
+	go-licenses save ./cli/ --save_path LICENSES; \
+	cp -R ./include/libbpf ./LICENSES
```

Why not?

```suggestion
	rm -rf LICENSES
	go-licenses save ./cli/ --save_path LICENSES
	cp -R ./include/libbpf ./LICENSES
```

Otherwise if `go-licenses save ./cli/ --save_path LICENSES` crashes then this will silently pass